cranelift-codegen/src/ir/entities.rs

```diff
@@ -122,7 +148,13 @@ impl Constant {
     }
 }
 
-/// An opaque reference to a jump table.
+/// An opaque reference to a [jump table](https://en.wikipedia.org/wiki/Branch_table).
+///
+/// `JumpTable`s are used for indirect branching.
+/// See [`br_table`](super::InstBuilder::br_table) for details.
+///
+/// `JumpTable`s can be created with
+/// [`create_jump_table`](cranelift_frontend::FunctionBuilder::create_jump_table).
```

Comment:
If you want a jump table which doesnt start at 0, or is not contiguous, you should use cranelift_frontend::Switch instead.